react-native/training/src/app/screens/ListBooks/components/Item/layout.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, Image, TouchableHighlight } from 'react-native';
+
+import CustomText from '../../../../components/CustomText';
+import defaultImage from '../../../../../assets/default-image-user.png';
+import { itemsBookList } from '../../../../../propTypes/propTypes';
+
+import styles from './styles';
+
+function Item({ data }) {
```

Comment:
..and I'd rename this file to `index.js`and convert `Item` comp to `PureComponent`